doc/source/math/direction.rst

```diff
@@ -211,31 +213,30 @@ Structure
     
     :struct: Gives a `Direction` with the opposite rotation around its axes.
     
-
-.. note:: **The difference between a :struct:`Direction` and a ``Vector``**
-
-    ``Vector`` and a :struct:`Direction` can be represented with the exact same amount of information: a tuple of 3 floating point numbers. So you might wonder why it is that a ``Vector`` can hold information about the magnitude of the line segment, while a :struct:`Direction` cannot, given that both have the same amount of information. The answer is that a :struct:`Direction` does contain one thing a ``Vector`` does not. A :struct:`Direction` knows which way is "up", while a ``Vector`` does not. If you tell **kOS** to ``LOCK STEERING`` to a ``Vector``, it will be able to point the nose of the vessel in the correct direction, but won't know which way you want the roof of the craft rotated to. This works fine for axial symmetrical rockets but can be a problem for airplanes.
-
```

Comment:
Hmm - I don't care so much about the ``..note`` markup as the loss of the title phrase "the difference between".  If that could be incorporated into the normal formatting of the titles for the sections that would be fine.